proto/ibc/lightclients/solomachine/v2/solomachine.proto

```diff
@@ -78,6 +78,22 @@ message TimestampedSignatureData {
   uint64 timestamp                   = 2;
 }
 
+// SignBytesV2 defines the signed bytes used for signature verification.
+message SignBytesV2 {
```

Comment:
we can drop the v2 and remove the old `SignBytes`

Reply:
Yup, will do. I'll probably try to remove the unused types first, merge the removal to this branch and then update the naming here on this PR.

Reply:
Removed the v2 suffixes